src/Api/Program.cs

```diff
@@ -95,11 +90,7 @@
                     configuration: ddConfig);
             }
         }
-    },
-    false,
-    // Pass log events down to other logging providers (e.g. Microsoft) after Serilog, so
-    // that they can be processed in a uniform way in tests.
-    true
```

Comment:
@jonashendrickx issue: This was added for a reason, if we just remove it we'll break logging in tests as indicated through the comment/git history.

My suggestion was that we set this argument depending on if we're running in env.Prod instead - was there an issue with my suggestion?

Reply:
I just couldn't read the suggestion in the chat - the thread refused to load.